keyboards/jaykeeb/jk65/info.json

```
@@ -0,0 +1,415 @@
+{
+    "manufacturer": "Jaykeeb Studio",
+    "keyboard_name": "JK65",
+    "maintainer": "Alabahuy",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["GP24", "GP14", "GP13", "GP12", "GP11", "GP10", "GP9", "GP8", "GP7", "GP6", "GP5", "GP4", "GP3", "GP2", "GP1"],
+        "rows": ["GP29", "GP0", "GP15", "GP26", "GP27"]
+    },
+    "indicators": {
+        "caps_lock": "GP25",
+        "on_state": 0
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x7765",
+        "vid": "0x414C"
+    },
+    "layout_aliases": {
+        "LAYOUT_65_ansi_blocker": "LAYOUT_ansi_blocker",
+        "LAYOUT_65_ansi_blocker_split_bs": "LAYOUT_ansi_blocker_split_bs",
+        "LAYOUT_65_ansi_blocker_tsangan": "LAYOUT_ansi_blocker_tsangan",
+        "LAYOUT_65_ansi_blocker_tsangan_split_bs": "LAYOUT_ansi_blocker_tsangan_split_bs"
+    },
```

- remove layout aliases
- replace with Community Layouts suport
```suggestion
    "community_layouts": ["65_ansi_blocker", "65_ansi_blocker_split_bs", "65_ansi_blocker_tsangan", "65_ansi_blocker_tsangan_split_bs"],
```